test/spec/provider/element-templates/cmd/ChangeElementTemplateHandler.spec.js

```diff
@@ -56,11 +56,12 @@ describe('element-templates - ChangeElementTemplateHandler', function() {
     container = TestContainer.get(this);
   });
 
-  function bootstrap(diagramXML) {
+  function bootstrap(diagramXML, elementTemplates) {
```

Comment:
Do we really need that? The other test cases come out pretty well without it

Reply:
The problem here is that due to changing the element Template, the `replaceBehavior` is triggered which will unlink the template unless it is present and retrievable from the registry https://github.com/bpmn-io/bpmn-js-properties-panel/blob/2eb11c9c20ad7f2d4f888a22cfb82341cc2468ad/src/provider/element-templates/ReplaceBehavior.js#L29